### PR TITLE
002 - explicit note about inner package behavior being preserved

### PR DESCRIPTION
**Pull request by bmeck**
this was discussed and intended, but stated poorly. explicit makes it easier to understand.
- thanks @ljharb for finding this, was not explicitly stated

### CODE REVIEW

File: 002-es6-modules.md [hunk 1]
```diff
@@ -199,11 +199,14 @@ ES `import` statements will not perform non-exact searches on relative or
...
-will continue to be unchanged and look to parent `node_modules` directories
-recursively as well.
+`node_modules` based paths (sometimes called "bare" paths) will continue to use 
+searching for both compatibility and to not limit the ability to have
+`package.json` support both ES and CJS entry points in a single
+codebase.`node_modules` based behavior will continue to be unchanged and look
+to parent `node_modules` directories recursively as well. This searching
+behavior explicitly includes inner package searching such ass `foo/bar`.
+
+Any entry into `node_modules` via paths not starting with "/", "./", or "../" will be using the same mechanism of searching while resolving their **entry point** as `require()`.
```
Comment (jasnell) [inline 002-es6-modules.md]: nit: line wrap

Comment (bmeck) [inline 002-es6-modules.md]: fixed, squashed into

**Comment by trevnorris** [comment]
LGTM

**Comment by trevnorris** [comment]
Thanks much! Landed in d354f89 and 4212287.

**Pull request closed without merging**